### PR TITLE
APM > Configuration > .NET > Adding IIS environment variables

### PR DESCRIPTION
**Pull request by zacharycmontoya**
### What does this PR do?
Adds steps to add environment variables to IIS sites. It also makes clear that environment variables added here apply to ALL IIS sites.

### Motivation
It's not clear / impossible for users to know how to update environment variables for IIS sites.

### Additional Notes
This might not be the best option for adding configuration, but I feel like we should at least demonstrate how to do it. Does the APM .NET team agree?

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/tracing/trace_collection/dd_libraries/dotnet-core.md [hunk 1]
```diff
@@ -354,6 +354,27 @@ Set-ItemProperty HKLM:SYSTEM\CurrentControlSet\Services\<SERVICE NAME> -Name Env
 
 {{< /tabs >}}
 
+#### IIS
+
+After installing the MSI, no additional configuration is needed to automatically instrument your IIS sites. If you'd like to set additional environment variables that will be inherited by all IIS sites, perform the following steps:
+
+1. Open the Registry Editor, find the multi-string value called `Environment` in the `HKLM\System\CurrentControlSet\Services\WAS` key, and add environment variables one-per-line.
```
Comment (lucaspimentel) [inline content/en/tracing/trace_collection/dd_libraries/dotnet-core.md:361]:
```suggestion
1. Open the Registry Editor, find the multi-string value called `Environment` in the `HKLM\System\CurrentControlSet\Services\WAS` key, and add the environment variables, one per line.
```

File: content/en/tracing/trace_collection/dd_libraries/dotnet-core.md [hunk 2]
```diff
@@ -354,6 +354,27 @@ Set-ItemProperty HKLM:SYSTEM\CurrentControlSet\Services\<SERVICE NAME> -Name Env
 
 {{< /tabs >}}
 
+#### IIS
+
+After installing the MSI, no additional configuration is needed to automatically instrument your IIS sites. If you'd like to set additional environment variables that will be inherited by all IIS sites, perform the following steps:
```
Comment (lucaspimentel) [inline content/en/tracing/trace_collection/dd_libraries/dotnet-core.md:359]:
```suggestion
After installing the MSI, no additional configuration is needed to automatically instrument your IIS sites. To set additional environment variables that are inherited by all IIS sites, perform the following steps: 
```

**Review by lucaspimentel — APPROVED** [review]

**Review by pierotibou — APPROVED** [review]
Good idea. Thanks for doing it. 
I can't wait to have remote config though :)

File: content/en/tracing/trace_collection/dd_libraries/dotnet-framework.md [hunk 3]
```diff
@@ -297,6 +297,27 @@ Set-ItemProperty HKLM:SYSTEM\CurrentControlSet\Services\<SERVICE NAME> -Name Env
...
+
+1. Open the Registry Editor, find the multi-string value called `Environment` in the `HKLM\System\CurrentControlSet\Services\WAS` key, and add the environment variables, one per line.
+2. Run the following commands to restart IIS:
+   ```cmd
+   net stop /y was
+   net start w3svc
+   # Also, start any other services that were stopped when WAS was shut down.
+   ```
+
+For example, to add logs injection and runtime metrics, add the following lines to the value data:
```
Comment (estherk15) [inline content/en/tracing/trace_collection/dd_libraries/dotnet-framework.md:312]: Same here, if this is related to step 1, could this be moved?

**Review by estherk15 — APPROVED** [review]

File: content/en/tracing/trace_collection/dd_libraries/dotnet-core.md [hunk 4]
```diff
@@ -354,6 +354,27 @@ Set-ItemProperty HKLM:SYSTEM\CurrentControlSet\Services\<SERVICE NAME> -Name Env
...
+
+1. Open the Registry Editor, find the multi-string value called `Environment` in the `HKLM\System\CurrentControlSet\Services\WAS` key, and add the environment variables, one per line.
+2. Run the following commands to restart IIS:
+   ```cmd
+   net stop /y was
+   net start w3svc
+   # Also, start any other services that were stopped when WAS was shut down.
+   ```
+
+For example, to add logs injection and runtime metrics, add the following lines to the value data:
```
[the remark this answers is not in the document]
Comment (zacharycmontoya) [reply content/en/tracing/trace_collection/dd_libraries/dotnet-core.md]: Yes it is, and yes I can move it under step 1. I've updated it for your feedback in dcebfcc6cb6642b47124189a96b29820dbe4fed9

File: content/en/tracing/trace_collection/dd_libraries/dotnet-framework.md (hunk 3 above)
Comment (zacharycmontoya) [reply content/en/tracing/trace_collection/dd_libraries/dotnet-framework.md]: Yes it is, and yes I can move it under step 1. I've updated it for your feedback in dcebfcc6cb6642b47124189a96b29820dbe4fed9

**Comment by zacharycmontoya** [comment]
I've added some edits in response to your feedback. Feel free to merge if it looks good 😄

**Comment by zacharycmontoya** [comment]
@estherk15 can we merge this?

**Pull request merged**